### ISSUE TITLE
Feature request:  Optional change on landing page

### ISSUE THREAD

**Issue by rsangole** [calintat/minimal#27]
Calin,

For the landing page, could we optionally add code that would give us something like:

   Title
Subtitle
(Posts in the past 7 days: x)

x can be a link to take us to posts or projects. The intent of this is to allow new visitors to quickly see if there's something new published recently...

Rahul

**Comment by jamestharpe** [comment]
You can copy index.html to your site's /layouts directory and change it there.

**Comment by rsangole** [comment]
Thanks, let me try that out. Any idea how I can get a count of the new posts in the past 7 days?

**Comment by jamestharpe** [comment]
You can [list pages](https://gohugo.io/templates/lists/) by date: 

```
{{ range .Data.Pages.ByDate }}
```

And use [where](https://gohugo.io/functions/where/) to filter the pages.

I can try to come up with a more complete example, but it might not be until next weekend that I have a chance to put it together.

**Comment by joshlawrence** [comment]
I'm having trouble with this as well, here's what I've done so far:

I copied index.html to my site's layouts directory.  I took the contents of the theme's list.html and put it in the index:

```
{{ partial "header" . }}
<main>
  <h2>{{ .Title }}</h2>
  {{ range (.Paginator 5).Pages }} {{ partial "list-item" . }} {{ end }}
</main>
{{ partial "paginator" . }}
{{ partial "footer" . }}
```

This works well - I get a list of posts.  But it _also_ grabs my pages (for example, about.md) and puts those in the list.  How do I avoid that?  I'm trying to get my index.html to behave exactly like the theme's Posts page...I'm close, but I'm missing something.

Wonderful work on this theme, I really do love it and it's very easy to work with...Thank you for working on it and making it available!
